### PR TITLE
Fix URL of ukri.org in README

### PR DESCRIPTION
**Pull request by jl-wynen**
See scipp/scippneutron#336

### CODE REVIEW

File: tests/dataset_test.py [hunk 1]
```diff
@@ -284,6 +284,7 @@ def test_coords_update_from_sequence_of_tuples_adds_items():
 
 
 def test_coords_update_from_iterable_of_tuples_adds_items():
+
```
Comment (nvaytet) [inline tests/dataset_test.py:287]: Strange unrelated change?

**Review by nvaytet — APPROVED** [review]

File: tests/dataset_test.py (hunk 1 above)
Comment (jl-wynen) [reply tests/dataset_test.py:287]: This showed up in a lot of PRs. It comes from the update of our yapf version.
